lighthouse-core/gather/gather-runner.js

```diff
@@ -135,7 +136,9 @@ class GatherRunner {
    * @param {!Array<WebInspector.NetworkRequest>} networkRecords
    */
   static assertPageLoaded(url, driver, networkRecords) {
-    const mainRecord = networkRecords.find(record => record.url === url);
+    const mainRecord = networkRecords.find(record => {
```

Comment:
maybe add a comment here that the network record never sees the hash so we need to test the URL without it. Obvious>